### PR TITLE
PopupView respects showAttachments and thumbnail size.

### PR DESCRIPTION
**Pull request by anmacdonald**
* PopupView now respects the `showAttachments` property.
* PopupView now respects the `thumbnailSize` property.

### CODE REVIEW

File: uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/PopupViewController.qml [hunk 1]
```diff
@@ -33,42 +33,60 @@ QtObject {
...
 
     /*!
       \brief Exposes the number of rows in the list model returned by 
       `attachments`.
-    */
+     */
     readonly property alias attachmentCount: internal.attachmentCount
 
+    /*!
+      \brief The mimimum attachment thumbnail width.
```
Comment (ldanzinger) [inline uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/PopupViewController.qml:65]:
```suggestion
      \brief The minimum attachment thumbnail width.
```

File: uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/PopupViewController.qml [hunk 2]
```diff
@@ -33,42 +33,60 @@ QtObject {
...
+     */
+    property alias attachmentThumbnailWidth: internal.attachmentThumbnailWidth
+
+    /*!
+      \brief The mimimum attachment thumbnail height.
```
Comment (ldanzinger) [inline uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/PopupViewController.qml:70]:
```suggestion
      \brief The minimum attachment thumbnail height.
```

**Review by ldanzinger — COMMENTED** [review]
@anmacdonald I was reading through the changes and spotted a few doc updates needed

**Comment by anmacdonald** [comment]
@ldanzinger good find!

File: uitools/cpp/Esri/ArcGISRuntime/Toolkit/PopupViewController.cpp [hunk 3]
```diff
@@ -158,6 +162,18 @@ int PopupViewController::attachmentCount() const
   return 0;
 }
 
+/*!
+  \brief Returns whether attachments should be displayed dor not
```
Comment (JamesMBallard) [inline uitools/cpp/Esri/ArcGISRuntime/Toolkit/PopupViewController.cpp:166]:
```suggestion
  \brief Returns whether attachments should be displayed or not
```

File: uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/PopupViewController.qml [hunk 4]
```diff
@@ -83,41 +101,63 @@ QtObject {
...
     onPopupManagerChanged: {
+        // PopupManager properties have no NOTIFY or CONST properties.
+        // To avoid property warnings we will instead cache 
+        // what comes out of PopupManager.
         if (popupManager) {
             if (popupManager.attachmentManager) {
                 internal.attachments = popupManager.attachmentManager.attachmentsModel;
                 internal.attachmentCount = attachments.rowCount();
+                internal.attachmentThumbnailWidth = Qt.binding(() => internal.attachments.thumbnailWidth);
+                internal.attachmentThumbnailHeight = Qt.binding(() => internal.attachments.thumbnailHeight);
```
Comment (JamesMBallard) [inline uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/PopupViewController.qml:140]: This is cool

**Review by JamesMBallard — APPROVED** [review]
Small doc typo, but no need to re-review.

**Pull request merged**